app/icon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
It can be extended with more `path`s in the `switch-case` block if wanted :) A SVG symbol sheet could be used instead, if we don't want a full React component for it. But I don't see any downside with it tbh.